keps/sig-scheduling/2926-job-mutable-scheduling-directives/README.md

```diff
@@ -574,7 +575,7 @@ that might indicate a serious problem?
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-It will be tested manually prior to beta launch.
+Tested manually.
```

Comment:
Can you describe the setup it was tested in? What exactly you were checking for? Any findings?

Reply:
Done.

Reply:
Note that this was tested back when we went to beta, the KEP was just not updated.